dist_run.py

```diff
@@ -110,26 +110,33 @@ def _build_chat_tokenizer(
     return tokenizer
 
 
-def _load_model_weights(stage_module, distribution, device, model_config):
+def _load_model_weights(
+    stage_module: torch.nn.Module,
+    distribution: str,
+    device: torch.device,
+    model_config: ModelArgs,
+    chpt_from: str,
```

Comment:
for better clarity, 'chkpt' is a much better abbreviation for checkpoint.  chpt is short for 'chapter' which is confusing.